apps/src/sharedFunctionalBlocks.js

```diff
@@ -348,7 +468,7 @@ function installCond(blockly, generator, numPairs) {
   generator[blockName] = function() {
     var cond, value, defaultValue;
     var code = 'function() {\n  ';
-    for (var i = 0; i < numPairs; i++) {
+    for (var i = 0; i <= this.numPairs_; i++) {
```

Comment:
Ah, could this just be `this.pairs_.length` to single-point-of-truth it? Or are they out of sync during a period?